tests/generator-Tests/Unit-Tests/CodeGeneratorTests.cs

```diff
@@ -352,6 +352,32 @@ public void SkipInvokerMethodsMetadata ()
 			Assert.False (writer.ToString ().Contains ("public abstract Com.Xamarin.Android.MyBaseClass DoStuff ();"), $"was: `{writer}`");
 		}
 
+		[Test]
+		public void SkipInterfaceInvokerMethodsMetadata ()
+		{
+			var xml = @"<api>
+			  <package name='java.lang' jni-name='java/lang'>
+			    <class abstract='false' deprecated='not deprecated' final='false' name='Object' static='false' visibility='public' jni-signature='Ljava/lang/Object;' />
+			  </package>
+			  <package name='com.xamarin.android' jni-name='com/xamarin/android'>
+			    <interface abstract='false' deprecated='not deprecated' extends='java.lang.Object' extends-generic-aware='java.lang.Object' jni-extends='Ljava/lang/Object;' final='false' name='MyInterface' static='false' visibility='public' jni-signature='Lcom/xamarin/android/MyInterface;' skipInvokerMethods='com/xamarin/android/MyInterface.countAffectedRows()I'>
```

Comment:
Generally (always?) `//interface/@abstract` will be `true`, not false.  (At least, I can find no instances of such a thing in [`api-34.xml`](https://github.com/xamarin/xamarin-android/blob/e41a31b9520dd1a4dbfe8162cf49fed7052dc69f/src/Mono.Android/Profiles/api-34.xml).